src/main/java/fr/sncf/osrd/speedcontroller/generators/MarecoAllowanceGenerator.java

```diff
@@ -139,20 +230,23 @@ private static TrainPositionTracker convertPosition(TrainSchedule schedule, Simu
         return location;
     }
 
+    @SuppressWarnings("checkstyle:WhitespaceAfter")
```

Comment:
I don't thing this warning should be suppressed. Line `245` there is a missing white space, maybe it's why the warning is trigger.

Reply:
Ok